### PR TITLE
Merging two Arrays

### PR DESCRIPTION
**Pull request by HariniJeyaraman**
**Fixes issue:**
 #3685

**Changes:**

 Implemented merging of 2 arrays in C++

### CODE REVIEW

**Comment by HariniJeyaraman** [comment]
@arnavb @AdiChat Please review the code for merging 2 arrays in C++

File: Merge_Arrays/merge_arrays.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#include<stdlib.h>
+#include<iostream>
+using namespace std;
```
Comment (arnavb) [inline Merge_Arrays/merge_arrays.cpp]: Don't use this.

File: Merge_Arrays/merge_arrays.cpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+void merge(int arr[], int arr1[], int arr2[], int n1, int n2)
+{
+    int k=0;
+    for(int i = 0; i < n1; i++)
```
Comment (arnavb) [inline Merge_Arrays/merge_arrays.cpp]: Use pre-increment.

File: Unique_Numbers/unique_number.cpp [hunk 3]
```diff
@@ -0,0 +1,40 @@
+#include<iostream>
```
Comment (arnavb) [inline Unique_Numbers/unique_number.cpp]: Please use different branches for your different pull requests.

File: Merge_Arrays/merge_arrays.cpp [hunk 4]
```diff
@@ -0,0 +1,39 @@
+#include<stdlib.h>
```
Comment (arnavb) [inline Merge_Arrays/merge_arrays.cpp]: Reformat this file using clang-format.

**Comment by arnavb** [comment]
Superseded by #4708.

**Pull request closed without merging**